content/10.zk-stack/20.running/55.ownership-model.md

```diff
@@ -0,0 +1,117 @@
+
+## Roles overview
+
+| Name | Purpose                                                                                                                                                                                                                 | Owned by | Shared | Source code                                                                                                                                             |
+| --- |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| --- | --- |---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **DecentralizedGovernance** (set of contracts) | Owns ProtocolUpgradeHandler, controls all ecosystem contracts, accepts/vetoes proposals.                                                                                                                                | - | Yes | [Repository](https://github.com/zksync-association/zk-governance/tree/96ade35e25e66d44af8736172c63aac3b5386150)                                         |
+| **ProtocolUpgradeHandler** | Ownership of CTM, Bridgehub, L1 and L2 bridges, AssetRouters, NTVs                                                                                                                                                      | DecentralizedGovernance | Yes | [Source](https://github.com/zksync-association/zk-governance/blob/96ade35e25e66d44af8736172c63aac3b5386150/l1-contracts/src/ProtocolUpgradeHandler.sol) |
+| **ML ChainAdmin** | - Revert batches within CTM<br/>- Set validator to any chain (will be removed in gateway upgrade)<br/>- Create new chain in Bridgehub<br/>- Add a base token in Bridgehub<br/>- Allow bridging to some L2SharedBridge   | ML Multisig | Yes | [Source](https://github.com/matter-labs/era-contracts/tree/0fcc77c4f49d10efaaba4808b2c9b29a6fbfb232/l1-contracts/contracts/governance/ChainAdmin.sol)   |
+| **ChainAdmin** | - Change fee parameters<br/>- Set validator for a chain<br/>- Set transaction filterer<br/>- Set base token multiplier<br/>- Set pubdata pricing mode<br/>- Set the timestamp when the protocol upgrade will be applied | Chain operator’s Multisig | No | [Source](https://github.com/matter-labs/era-contracts/tree/0fcc77c4f49d10efaaba4808b2c9b29a6fbfb232/l1-contracts/contracts/governance/ChainAdmin.sol)   |
```

Comment:
better to use this one: https://github.com/matter-labs/era-contracts/blob/0fcc77c4f49d10efaaba4808b2c9b29a6fbfb232/l1-contracts/contracts/governance/ChainAdminOwnable.sol